.github/workflows/awie.yml

```
@@ -87,6 +87,7 @@ jobs:
           release: ${{ needs.get-version.outputs.release }}
           nexus_username: ${{ secrets.ARTI_USER }}
           nexus_password: ${{ secrets.ARTI_PASSWD }}
+          arti_token: ${{ secrets.ARTI_APT_TEST_TOKEN }}
```

same suggestion for others :)
```suggestion
          artifactory_token: ${{ secrets.ARTI_APT_TEST_TOKEN }}
```